pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -2550,8 +2550,8 @@ self: super: {
 
   # attoparsec bump is on v2 branch, but not released yet
   irc-core = assert super.irc-core.version == "2.10"; doJailbreak super.irc-core;
-  glirc = assert super.irc-core.version == "2.10"; doJailbreak super.glirc;
-  hookup = assert super.irc-core.version == "2.10"; doJailbreak super.hookup;
+  # glirc 2.39 is broken, downgraded to 2.38 to unbreak
+  glirc = assert super.irc-core.version == "2.10"; doJailbreak (markUnbroken super.glirc);
```

Comment:
Just delete `glirc` from `broken.yaml` instead and regenerate the package set.

Reply:
I tried that first, but that still pulled in glirc 2.39 and marked it as not broken. Is that still preferable?

Reply:
I've pushed a commit that removes glirc from `broken.yaml`. Now `haskellPackages.glirc_2_39` is not marked as broken, but fails to build. Should I mark this broken in `configuration-common.nix` then?

Reply:
Right, you need to add the default-package-overrides as well of course.

Reply:
Oh that was what you meant! Don't bother, we don't give any guarantees over the versioned attributes (they are not built usually), so it's not required to mark them correctly!